### DISCUSSION TITLE
SCSS style preprocessor

### DISCUSSION THREAD

**Discussion by FlexonGordon** [brackets-cont/brackets#314] [Q&A]
How to add SCSS style preprocessor to brackets?
I can’t find information anywhere, I don’t know myself, I just started studying

**Comment by charlypa** [comment]
hope this video helps you
https://www.youtube.com/watch?v=cEiORP1uG5I&t=9s

**Comment by charlypa** [comment]
@FlexonGordon 
We have modernized brackets. We released it yesterday. The new editor is Phoenix Code, Now you can download new installers instead of relying on old Brackets  binaries
visit https://phcode.io/
